### ISSUE TITLE
Errors in MStore Pro 4.0.0

### ISSUE THREAD

**Issue by codigojp2020** [inspireui/support#4386]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
NOTE: The following errors occur when debugging the source code version 4.0.0

    1.- Error with Login with SMS Firebase, this error occurs when applying the verification code.

    2.- Design errors:
          - The background color of the ToolBar does not look transparent like in the DEMO.
          - The Menus of the Modal Menu they don't look very good.

    3.- In Versions of Android 9 or higher does not show any product

+ Product & version: MStore Pro 4.0.0
+ Flutter (or React Native) version: 0.62.2
+ Testing Device/Simulator: Huawei P10 Lite(Android=8.0), Samsung S10 (Android=10.0), Samsung A10(Android=9.0)
+ Screenshot issues (drag the file to attach here): 
![a](https://user-images.githubusercontent.com/68796504/88577671-68d68000-d015-11ea-8659-156fc5dca7f5.jpg)
![a1](https://user-images.githubusercontent.com/68796504/88577678-6bd17080-d015-11ea-82e9-c11efcaf7346.jpg)
![b](https://user-images.githubusercontent.com/68796504/88577682-6d029d80-d015-11ea-85db-c0c6b7fd1eca.jpg)
![C](https://user-images.githubusercontent.com/68796504/88577689-6f64f780-d015-11ea-9af0-b745767eee2e.jpg)

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by ameenbadri7** [comment]
Thank you for the feedback.

Please help us to submit this form and fill all required info to continue with the support - http://verify.inspireui.com/ - please don't share the purchase code or screenshot public here as another user will use it for activation.

We will close this ticket in 24 hours if it is not provided enough detail, kindly reopen the ticket after you have done the form submission. 

Thank you for your time!

**Comment by codigojp2020** [comment]
Hello @ameenbadri7 , I already sent the form that they request with all the data. help me please.

**Comment by luyx2412** [comment]
Hi @codigojp2020 Can i know the Mstore version Pro or expo? Because i just tested and the UI still work normal.
![image](https://user-images.githubusercontent.com/8909619/89094127-0f8e9800-d3eb-11ea-9852-3a9ebd11a157.png)

And please send me the log issue about login sms. Let me investigate about it.

Thanks

**Comment by codigojp2020** [comment]
Hello @luyx2412 , I already deleted the login button with sms because I did not find a solution, also in the demo, the login with SMS does not work either.

there I sent you the error "verify fail"

The UI DOES NOT WORK correctly when running the code in debug mode and release mode
- The toolbar appears with a white background when doing a scrollview up. in the demo it has a transparent background.
why does that happen?

I am using Mstore PRO 4.0.0

previously I already sent you in a screenshot the errors that I get
